### ISSUE TITLE
FE: Add VA health chat and VA health connect to the Manage your health online box

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#17436]
## Description
Veterans need to be able to discover alternate ways to communicate with their health care teams and complete tasks, such as managing appointments and prescriptions. 
The Facilities team needs a way to assess where Veterans are most likely to look for and learn about these options. 

We're making a bet that by adding VA health chat and VA health connect to the system level Manage your health online box, we can increase the click through rate from VAMC pages to these information pages. This bet makes sense because 

- these tools can be used to complete tasks similar to those included in this area
- the Manage your health online box is hard coded on the front end and therefore can be added centrally to all pages, rather than requiring editor action
- we can assess this option compared to program pages and spotlights created by editors and currently in the wild 

This should be done during the same sprint as [Update VA health connect phone number label #17750](https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/17750), if possible

Relevant issue: [#17308 Design: Add VA health connect and VA health chat to VAMC page](https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/17308)

### New design
<img width="375" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK0RNQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--e6e51acf421a2833849ccd1927102a741dd95f1c/Screenshot%202024-07-10%20at%201.56.14%20PM.png" alt="Screenshot 2024-07-10 at 1.56.14 PM.png" />

## Exception
This does not apply to Lovell TRICARE.

## Tasks

Based on the [Figma design](https://www.figma.com/design/ikCp2eOubIgbN1TwyEYIrN/VAMC-Master?node-id=92-7706&t=HaIDJCZHww8rSSqB-1), 
### Add VA Health Chat
- [ ] add 'chat' USWDS v3 icon 
- [ ] include `aria-hidden="true"` on icon
- [ ] add link text "Download VA Health Chat" with a link [https://mobile.va.gov/app/va-health-chat]
### Add VA Health connect 
- [ ] add 'phone' USWDS v3 icon 
- [ ] include `aria-hidden="true"` on icon
- [ ] add link text "Connect to VA care" with a link https://www.va.gov/health/connect-to-va-care/index.asp

### Additional Acceptance Criteria
- [ ] all existing links remain
- [ ] new links should appear below existing links
- [ ] Change header to h2 (currently h3)

**Comment by xiongjaneg** [comment]
Michelle and Jane to check on CLP status.
If needed, Michelle to check on interim link target if not CLP.

**Comment by xiongjaneg** [comment]
@laflannery I closed this ticket without saving your comment about the aria label needed for these old icons. Would you please add to the ACs? Thank you!

**Comment by laflannery** [comment]
Done!

**Comment by aklausmeier** [comment]
I am confirming with design system team, exact icons to use. We are in between current icons and moving to USWDS, will update the design and here when confirmed. https://dsva.slack.com/archives/C01DBGX4P45/p1709832068721499

**Comment by aklausmeier** [comment]
@xiongjaneg Confirmed with [design system folks](https://dsva.slack.com/archives/C01DBGX4P45/p1709832068721499), as part of this work we will need to update all icons in the "Manage your health ..." section to use USWDS icons (v3). 

I have updated my designs to reflect this. Screenshot attached for reference. 

![Screenshot 2024-03-08 at 12 54 22 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/137226108/123ffab2-472f-43dc-bb55-3c7eef913375)

**Comment by aklausmeier** [comment]
I have created a ticket with the design system team to add the icon for VA Health Chat app as we need one that was distinct from Secure messaging. 
https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2573

**Comment by mmiddaugh** [comment]
Confirmed with Lovell:  we will not need to promote VA health chat or VA health connect on the Tricare experience

**Comment by mmiddaugh** [comment]
This should be prioritized during Sprint 2, which begins 4/17/24

**Comment by aklausmeier** [comment]
@mmiddaugh Do we have a published CLP url?

**Comment by mmiddaugh** [comment]
This work is blocked pending VHA Comms strategy @jilladams @davidmpickett @aklausmeier FYI

**Comment by laflannery** [comment]
[New ticket to update Health connect phone number](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18246) which will also need to happen along with this ticket

**Comment by mmiddaugh** [comment]
This work is unblocked and should be prioritized in the next sprint @jilladams @agile6MSkinner @FranECross

**Comment by jilladams** [comment]
@Agile6MSkinner @FranECross for context: this was a Q2 priority initiative to add a single link to some of our pages, to point people to a chatbot interaction around VA Health. We did designs and research, but the initiative was paused temporarily while 2 different VA program teams aligned on how to talk about their programs. Now they're aligned and we can add the link.

The "assign PMs by initiative" model does leave gaps for things like this. This was originally a Facilities portfolio project, so I leave it to Fran / Michael to decide how y'all would like to own. There will be some comms to do to let VHA Digital Media know this has shipped, once it's out and verified.

FE folks pointed this as a 2 long ago - we can revisit that pointing during planning, or if someone is refining async with FE folks, Eli can sanity check it. (Chris / Randi won't have any context for it.)

**Comment by jilladams** [comment]
Michael agreed to own finishing this out from a PM perspective.

**Comment by laflannery** [comment]
Noting that I have [a heading level defect](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12079) for this block and it was suggested these tickets could be done at the same time

**Comment by mmiddaugh** [comment]
https://www.va.gov/health/connect-to-va-care/ is live as of 7/11/2024

**Comment by mmiddaugh** [comment]
Confirmed in Prod: <img width="825" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOXJqQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1122f8a32e87a369b3d54f0aee3a09756c46c476/image.png" alt="image.png" />

**Comment by chriskim2311** [comment]
Closing confirmed in Prod!
